source/development/contributing.rst

```diff
@@ -519,8 +550,9 @@ is added it is appended at the end of the file.
 Branch Protection
 *****************
 
-All repositories' ``main`` branch must be configured with a set of protection
-rules that aim at ensuring some of the rules defined in :ref:`contribution
+All repositories' ``main`` and ``wip/`` (base branch of a split of PRs)
+branches must be configured with a set of protection rules that aim at ensuring
+some of the rules defined in :ref:`contribution
```

Comment:
I think the "Splitting PRs section" should make a forward reference to this section.

Reply:
Fixed.